openspp-project/openspp-registry
# Hide Menu Module

## **Why is this change needed?**
To have a common and separated module for hiding of menus.

## **How was the change implemented?**
Created a module that can configure menus to be hidden.

## **New unit tests**
```
None
```

## **Unit tests executed by the author**
```
None
```

## **How to test manually**
- Install `spp_hide_menus`.
- (In Developer Mode) Go to Settings > User & Companies > Hidden Menus.
- Check if pre-created Menus are added in the List and also hidden in the Menu List.
- Check Users (Settings > User & Companies > Users) if `Hide Menu` is gone and replaced by `Show Non-OpenSPP Menus`.
- Try checking and unchecking this for a user to check if the expected behavior is correct.

## **Related links**
https://github.com/OpenSPP/openspp-modules/issues/633

##**PR Depend on this***
https://github.com/newlogic/openspp-lao-pdr/pull/70
